src/cryptonote_core/service_node_rules.h

```diff
@@ -143,7 +147,8 @@ namespace service_nodes {
     return
         hf_version <= cryptonote::network_version_12_checkpointing ? quorum_type::obligations :
         hf_version <  cryptonote::network_version_14_blink         ? quorum_type::checkpointing :
-        quorum_type::blink;
+        hf_version <  HF_VERSION_PULSE                             ? quorum_type::blink :
```

Comment:
Can this just be `cryptonote::network_version_16_pulse` to match the others?  (Since pulse seems the biggest feature, it seems more appropriate to embed that in the name and let the macros be used for ancillary features like `HF_VERSION_REJECT_SIGS_IN_COINBASE`).